src/app/views/scorecards/temp-extract.js

```diff
@@ -0,0 +1,77 @@
+define([], function () {
+  function getExtract() {
+    return {
+      property_type: 'Mixed Use Property',
+      id: '525',
+      xepaid: '1854824',
+      year: '2023',
+      property_name: 'SEATTLE ART MUSEUM',
+      reported_address: '1300 1ST AVE',
+      city: 'SEATTLE',
+      state: 'WA',
+      zip: '98101',
+      latitude: '47.6074',
+      longitude: '-122.33777',
+      neighborhood: 'DOWNTOWN',
+      councildistrict: '7',
+      yearbuilt: '1991',
+      numfloors: '6',
+      numbuildings: '1',
+      PropertyGFATotal: '280230',
+      'PropertyGFABuilding(s)': '280230',
+      PropertyGFAParking: '0',
+      SelfReportGFATotal: '539630',
+      'SelfReportGFABuilding(s)': '539630',
+      SelfReportParking: '0',
+      EPAParticipationStatusFlag: 'Y',
+      PrimaryPropertyType: 'Mixed Use Property',
+      // Made this up
+      energy_star_score: '55',
+      site_eui_wn: '75.90000153',
+      site_eui: '75.90000153',
+      total_kbtu: '40939768',
+      source_eui_wn: '161.3999939',
+      source_eui: '161.3999939',
+      electricity: '6977690',
+      steam: '16084122',
+      gas: '10478',
+      EPAPropertyType: 'Mixed Use Property',
+      IsCompliantFlag: 'Y',
+      total_ghg_emissions: '1075.4',
+      total_ghg_emissions_intensity: '3.84',
+      CBPSEUItarget: '68.9',
+      All_EPA_Space_Types: 'Museum, Office, Restaurant',
+      BEPStarget_2031: '1.52',
+      BEPStarget_2036: '0.89',
+      BEPStarget_2041: '0',
+      BEPStarget_2046: '0',
+      BEPS_FirstComplianceYear: '2031',
+      building_type_eui: '85.6',
+      building_type_eui_wn: '85.2',
+      electricity_pct: '0.58',
+      gas_pct: '0.03',
+      steam_pct: '0.39',
+      electricity_ghg: '138.09',
+      gas_ghg: '84.87',
+      steam_ghg: '852.46',
+      electricity_ghg_percent: '0.13',
+      gas_ghg_percent: '0.08',
+      steam_ghg_percent: '0.79',
+      cos_median_eui: '36.40000153',
+      percent_from_median: '0.11',
+      higher_or_lower: 'lower',
+      numunits: '0',
+      yearbuilt_string: '1991',
+      ess_cert: '0',
+      cost_annual: '952197',
+      cost_sq_ft: '3.397913885',
+      percent_save: '0.1',
+      amount_save: '95220',
+      comments: '',
+      pct_sum: '1',
+      uniqueid: '2023-525'
+    };
+  }
+
+  return getExtract;
+});
```

Comment:
Currently I'm just developing on the Seattle Art Museum to make things simpler, but will update this if needed depending on when real data comes online.